docs/product/issues/issue-details/index.mdx

```diff
@@ -102,6 +102,28 @@ If you’ve enabled [Session Replay](/product/explore/session-replay/), you’ll
 
 ![Session Replay](./img/issue-replay.png)
 
+## Feature Flags
+If you’ve enabled a feature flag integration in your SDK you’ll have access to the feature flag table. If any flags were captured after the Sentry SDK's initialization and before the error event occurred, the table will contain rows of flag, evaluation-result pairs.
+
+![Feature Flags](./img/feature-flags.png)
+
+A timeline of feature flag changes is displayed in the releases graph. These annotations can help you diagnose when a feature flag change is responsible for a new bug report.
+
+![Feature Flags](./img/suspect-flags-timeline.png)
+
+Additionally, feature flag definition changes can be flagged as suspect when their modification occurs suspiciously close to the creation of an error event. Suspect feature flags are displayed are given a yellow highlight within the feature flag table.
+
+![Feature Flags](./img/suspect-flags-table.png)
+
+To enable suspect feature flags and feature flag change annotations, click the "Set Up Integration" button on the feature flag table and follow the instructions. Provider specific instructions are provided by clicking the "Read Docs" button.
+
+![Feature Flags](./img/feature-flag-modal.png)
+
+To learn more about enabling feature flag change tracking, visit the documentation for one of the supported providers below.
+
+* [LaunchDarkly](/organization/integrations/feature-flag/launchdarkly/)
```

Comment:
This is docs for "suspect tags" integration on the launch-darkly website.

We also need to link to the "Sentry SDK" config, which confusingly also uses the name `LaunchDarklyIntegration`. Those docs are at https://docs.sentry.io/platforms/python/integrations/launchdarkly/

Reply:
Updated the link to point to the "Change Tracking Integration" heading on that page.